crates/issue/src/migration.rs

```diff
@@ -0,0 +1,209 @@
+use super::*;
+use frame_support::{pallet_prelude::*, traits::OnRuntimeUpgrade};
+use sp_core::H256;
+
+/// The log target.
+const TARGET: &'static str = "runtime::issue::migration::v1";
+
+/// The original data layout of the democracy pallet without a specific version number.
+mod v0 {
+    use super::*;
+
+    // Due to a known bug in serde we need to specify how u128 is (de)serialized.
```

Comment:
Does this comment still apply? I can't find this deserialization configured anywhere

Reply:
Ah it's because the v0 fields are annotated with macros